_posts/0000-01-02-Juha02.md

```diff
@@ -0,0 +1,5 @@
+---
+layout:slide
+title: "Welcome to our new slide!"
+---
+Hello
```

Comment:
God, grant me the serenity to accept the things I cannot change, courage to change the things I can, and wisdom to know the difference.